docs/ipv6.md

```diff
@@ -1,114 +1,175 @@
+kube-router maintainers. Currently, we are targeting a v2.0.0 release of kube-router which will see all controllers
+updated for dual-stack compatibility.
 
-Test and development has so far used
-https://github.com/Nordix/xcluster/tree/master/ovl/kube-router-ipv6
-which is an easy way to get a ipv6-only Kubernetes cluster.
+We are currently running this work off of the
+[prep-v2.0 branch](https://github.com/cloudnativelabs/kube-router/tree/prep-v2.0) and, as of the time of this writing,
+have released a [release candidate]() with this some dual-stack functionality built into it.
 
-To setup an ipv6-only Kubernetes cluster is usually no simple task,
-see for instance https://github.com/leblancd/kube-v6
+Functions that currently support dual-stack on the v2.0.0 release line:
 
-No automatic tests exist yet for ipv6.
+* CNI
+* Router / BGP (`--run-router`)
+* Network Policies (`--run-firewall`)
 
+## How Can You Help?
 
-## Current status (Thu Oct 11 2018)
+The work to make dual-stack support workable in kube-router has involved a lot of work and several large contributions
+to the code base. If you have access to a development or staging environment where you would be willing to run the
+latest v2.0.0 release candidate and give us feedback, it would be much appreciated!
 
-Support for ipv6 in the the CNI function in `kube-router` is under
-development. The local BGP routers peers with ipv6;
+While the most helpful feedback will come from users that are able to run kube-router in dual-stack mode, there have
+been enough changes, that even users that are only able to run kube-router in a single-stack mode will be able to give
+valuable feedback concerning any bugs or regressions.
 
-```
-# gobgp neighbor
-Peer                AS  Up/Down State       |#Received  Accepted
-1000::1:c0a8:101 64512 00:00:37 Establ      |        0         0
-1000::1:c0a8:102 64512 00:00:37 Establ      |        0         0
-1000::1:c0a8:103 64512 00:00:40 Establ      |        0         0
-```
+If any users that find bugs have Golang experience, please consider helping us squash some of our dual-stack related
+bugs by filing PRs against the project.
 
-The CNI configuration is also updated with ipv6 addresses;
+All PRs related to the new v2.0.0-based functionality will be given priority in terms of review as we work to get this
+released and out into the wild so that we can work on other features and issues.
 
-```
-# jq . < /etc/cni/net.d/10-kuberouter.conf  | cat
-{
-  "bridge": "kube-bridge",
-  "ipam": {
-    "subnet": "1000::2:b00:100/120",
-    "type": "host-local"
-  },
-  "isDefaultGateway": true,
-  "isGateway": true,
-  "name": "ekvm",
-  "type": "bridge"
-}
-```
+## Roadmap
 
-This means that pod's gets assigned ipv6 addresses.
+The next big item to add to make kube-router fully dual-stack functional will be the **Proxy (`--run-service-proxy`)**
```

Comment:
Can we create an issue for this and reference it here so users can track it?

Reply:
Good idea! I've added a new issue template in this PR.

Reply:
I was thinking about creating an issue for the missing --run-service-proxy functionality, that users can track

Reply:
Ahh... I see now that I originally misunderstood this.

Here is a tracking issue: https://github.com/cloudnativelabs/kube-router/issues/1432